docs/src/pages/components/menus/menus.md

```diff
@@ -76,3 +76,11 @@ For more advanced use cases you might be able to take advantage of:
 There is a 3rd party package [`material-ui-popup-state`](https://github.com/jcoreio/material-ui-popup-state) that takes care of menu state for you in most cases.
 
 {{"demo": "pages/components/menus/MenuPopupState.js"}}
```

Comment:
You can use this syntax and then create e.g. `pages/components/menus/NavMenu.js` to create a full fledged demo. This helps users (and maintainers) audit how a11y works for nav menus.

You can look at the other menu demos how they're constructed.

Reply:
@eps1lon  That's definitely a good idea, is it possible, thought, to create a demon with `react-router-dom` within ie. `{{"demo": "pages/components/menus/MenuItemLink.js"}}`?

Reply:
Yes, we already do this in a couple of places e.g. https://github.com/mui-org/material-ui/blob/6159993a1cdb292984ac9fdb7f4a34759d51003a/docs/src/pages/components/buttons/ButtonRouter.js#L14

Reply:
@eps1lon That's great, will amend this evening 👍